Explorer/Assets/DCL/SDKComponents/AudioSources/AudioUtils.cs

```diff
@@ -1,11 +1,47 @@
-﻿using DCL.Diagnostics;
+﻿using Arch.Core;
+using CommunicationData.URLHelpers;
+using DCL.Diagnostics;
+using DCL.Optimization.Pools;
+using ECS.Prioritization.Components;
+using ECS.StreamableLoading.AudioClips;
+using ECS.StreamableLoading.Cache;
+using ECS.StreamableLoading.Common.Components;
+using SceneRunner.Scene;
 using System;
 using UnityEngine;
+using Promise = ECS.StreamableLoading.Common.AssetPromise<UnityEngine.AudioClip, ECS.StreamableLoading.AudioClips.GetAudioClipIntention>;
 
 namespace DCL.SDKComponents.AudioSources
 {
     public static class AudioUtils
     {
+        public static void CleanUp(this AudioSourceComponent component, World world, IStreamableCache<AudioClip, GetAudioClipIntention> cache, IComponentPool componentPool)
```

Comment:
You should pass `AudioSourceComponent` by ref, otherwise changes made in `ForgetLoading` won't be reflected in the original component.